docs/modules/ROOT/pages/mq_services/message_queue_system.adoc

```diff
@@ -23,25 +25,68 @@ NOTE: This document is under development
 
 [#system-requirements]
 == System Requirements
+1.	DPM can split the network configuration into multiple goal states. DPM can appropriately choose MQ or GRPC channel to send goal state to the corresponding ACA.
+2.	When ACA configures goal state successfully, it returns success reply. When DPM receives replies of all goal states for a network configuration, it returns successful configuration to the caller
+3.	Messaging services can support 1000,000 data plane nodes.
+4.	Working together with other services including Node, Port, VPC, Data Plane managers.
+5.	Fast, reliable and scalable
 
-=== Must-have features
+== Design
+=== Architecture
+image::MessageArchitecture.png[]
 
-=== Nice-to-have features
+We have two architectural design options.
+
+==== Design Option: MQ (Node Group Topic)
+image::file.png[]
 
-=== Throughput and Latency Requirements
+==== Design Option: MQ (VPC Topic) + GRPC
+===== Workflow
+image::vpcGroup.png[]
 
-== Review of Popular MQ Systems
+===== Workflow UML
+image::MQportcreate.png[]
 
+===== MQ topic subscription or unsubscription
+image::MQsubscribe.png[]
```

Comment:
This image appears to very large. Can you check Can you check https://github.com/issacyxw/alcor/blob/master/docs/modules/ROOT/pages/mq_services/message_queue_system.adoc and adjust the size?